src/Fixer/Basic/BracesFixer.php

```diff
@@ -478,7 +482,13 @@ function ($item) {
 
                 if (!$isAnonymousClass && $prevToken->isWhitespace() && false !== strpos($prevToken->getContent(), "\n")) {
                     if (!$tokens[$startBraceIndex - 2]->isComment()) {
-                        $tokens->ensureWhitespaceAtIndex($startBraceIndex - 1, 1, ' ');
+                        $tokens->ensureWhitespaceAtIndex(
+                            $startBraceIndex - 1,
+                            1,
+                            self::LINE_NEXT === $this->configuration['position_after_return_type_hint'] && $token->isGivenKind(T_FUNCTION) && ':' === $tokens[$tokens->getNextNonWhitespace($closingParenthesisIndex)]->getContent() ?
```

Comment:
`->equals(':')` doesn't appear to be doing the same thing... possibly something to do with `->isArray` setting?